### PR TITLE
Fix suggestion to use expand_cxxstring_abis

### PR DESCRIPTION
**Pull request by maleadt**
Ref https://github.com/JuliaPackaging/BinaryBuilder.jl/commit/491e6ccd723aa2c864b88f4670e3f1c3e0300ad2

### CODE REVIEW

File: src/auditor/compiler_abi.jl [hunk 1]
```diff
@@ -51,7 +51,8 @@ function check_libgfortran_version(oh::ObjectHandle, platform::Platform; verbose
 end
 
 """
-    detect_libstdcxx_version(oh::ObjectHandle, platform::Platform)
+    detect_libstd_ver
```
Comment (giordano) [inline src/auditor/compiler_abi.jl]: Extra new line

File: src/auditor/compiler_abi.jl [hunk 2]
```diff
@@ -185,7 +186,7 @@ function check_cxxstring_abi(oh::ObjectHandle, platform::Platform; io::IO = stdo
         both GCC 4 and GCC 5.  To do this, immediately after your `products`
         definition in your `build_tarballs.jl` file, add the line:
         """, '\n' => ' '))
-        msg *= "\n\n    products = expand_cxx_versions(products)"
+        msg *= "\n\n    products = expand_cxxstring_abis(products)"
```
Comment (giordano) [inline src/auditor/compiler_abi.jl]: The whole message is wrong, not just this line.  `expand_cxxstring_abis` applies to platforms, not products:
```suggestion
        msg *= "\n\n    platforms = expand_cxxstring_abis(platform)"
```
See https://github.com/JuliaPackaging/Yggdrasil/pull/393#discussion_r367450854

File: src/auditor/compiler_abi.jl (hunk 1 above)
Comment (maleadt) [reply src/auditor/compiler_abi.jl]: That will teach me for using the GitHub web interface...

File: src/auditor/compiler_abi.jl [hunk 3]
```diff
@@ -185,7 +185,7 @@ function check_cxxstring_abi(oh::ObjectHandle, platform::Platform; io::IO = stdo
         both GCC 4 and GCC 5.  To do this, immediately after your `products`
         definition in your `build_tarballs.jl` file, add the line:
         """, '\n' => ' '))
-        msg *= "\n\n    products = expand_cxx_versions(products)"
+        msg *= "\n\n    platforms = expand_cxxstring_abis(platform)"
```
Comment (giordano) [inline src/auditor/compiler_abi.jl]: Sorry, I'm apparently unable to type:
```suggestion
        msg *= "\n\n    platforms = expand_cxxstring_abis(platforms)"
```
Also the rest of the message needs to be adapted as it mentions products

**Comment by staticfloat** [comment]
Thanks Tim!

**Pull request merged**